etc/kayobe/ansible/wazuh-manager.yml

```diff
@@ -125,3 +115,40 @@
       service:
         name: wazuh-dashboard
         state: restarted
+
+- hosts: wazuh-agent
+  become: yes
+  become_user: root
+  tasks:
+    - name: Check if custom SCA policies directory exists
+      stat:
+        path: "{{ local_custom_sca_policies_path }}"
+      register: custom_sca_policies_folder
+      delegate_to: localhost
+      become: no
+
+    - name: Gather list of custom SCA policies
```

Comment:
Why is this run as root but the last task wasn't?

Reply:
I just missed a become: no here tbh. Have updated the becomes now